### PR TITLE
Py3

### PR DESCRIPTION
**Pull request by ghost**
Tests pass, looks like the only real issue was `map` being a generator in Py2.

### CODE REVIEW

**Comment by redapple** [comment]
Cool! Can you update the travis config to run Py3 tests?
Example matrix config: https://github.com/scrapy/scrapy/blob/master/.travis.yml#L8

**Comment by ghost** [comment]
I modified the `tox.ini` file, will that suffice? :)

**Comment by ghost** [comment]
Derp, I clearly don't grok travis yet.

File: tox.ini [hunk 1]
```diff
@@ -1,6 +1,3 @@
-[tox]
```
Comment (redapple) [inline tox.ini]: Does removing this affect tests run locally?

**Comment by ghost** [comment]
@redapple No. It's not ideal, though, because it means you can't run multiple version checks with one invocation of `tox`. The way I had it at first, running `tox` would check both versions at once. But that didn't play well with Travis.

To test locally now, you'd do `python{x} -m tox` to test Python x.

**Comment by redapple** [comment]
In other projects, we've been using `TOXENV` in the matrix in Travis and it's been working well.
And local tests still use `tox -e py27,py35` for example.
See https://github.com/scrapinghub/extruct/blob/master/.travis.yml for example

**Review by redapple — APPROVED** [review]
Looks good to me. Thank you @cathalgarvey

**Pull request closed without merging**